README.md

```diff
@@ -30,7 +37,7 @@ module load payu/1.1
 
```

Comment:
Should either update the `payu` version or just use the default (which should be latest)

Reply:
I've updated the payu version to 1.1.3. I think I am going to rewriting this readme anyway soon depending on the outcome of this issue: https://github.com/ACCESS-NRI/reproducibility/issues/27